code/datums/uplink_item.dm

```diff
@@ -697,3 +690,71 @@ var/list/uplink_items = list()
 	for(var/item in bought_items)
 		new item(C)
 		U.purchase_log += "<BIG>\icon[item]</BIG>"
+
+// JOB-SPECIFIC
+
+/datum/uplink_item/job_medsci
+	category = "Job-Specific Items: Medical/Science"
+	surplus = 0
+
+/datum/uplink_item/job_medsci/armored_labcoat
+	name = "Armored Labcoat"
+	desc = "A standard labcoat made of prototype materials embedded with chamelium. It functions identically to a regular labcoat but is more protective.\n\nJobs: All medical staff, all science staff"
+	cost = 3
+	item = /obj/item/clothing/suit/toggle/labcoat/syndicate
+	jobs = list("Medical Doctor", "Chemist", "Geneticist", "Chief Medical Officer", "Virologist", "Scientist", "Research Director", "Roboticist")
+
+/datum/uplink_item/job_medsci/stim_medipen
+	name = "Syndicate Medipen"
+	desc = "A red-and-black medipen loaded with a large recuperative cocktail. It still has the drawbacks of a regular medipen in that it cannot be refilled. Jobs: All medical staff"
+	cost = 10
+	item = /obj/item/weapon/reagent_containers/hypospray/medipen/syndicate
+	jobs = list("Medical Doctor", "Chemist", "Geneticist", "Virologist", "Chief Medical Officer")
+
+/datum/uplink_item/job_medsci/rad_laser
+	name = "Radioactive Microlaser"
+	desc = "A radioactive microlaser disguised as a standard Nanotrasen health analyzer. When used, it emits a powerful burst of radiation, which, after a short delay, can incapitate all but the most protected of humanoids. \
+	It has two settings: intensity, which controls the power of the radiation, and wavelength, which controls how long the radiation delay is. Jobs: All medical staff"
+	item = /obj/item/device/rad_laser
+	cost = 6
+	jobs = list("Medical Doctor", "Chemist", "Geneticist", "Virologist", "Chief Medical Officer")
+
+/datum/uplink_item/job_medsci/traitor_chem_bottle
+	name = "Poison Kit"
+	desc = "An assortment of deadly chemical solutions. Comes with a large variety, each fitting into their own niche. Jobs: Chemist"
+	item = /obj/item/weapon/storage/box/syndie_kit/chemical
+	cost = 6
+	jobs = list("Chemist")
+
+/datum/uplink_item/job_medsci/what_have_i_done
+	name = "Chlorine Trifluoride Foam Grenade"
+	desc = "A terrifying grenade that causes a gigantic area to be blanketed in chlorine trifluoride, designed to exterminate life. It is highly recommended you flee the scene before it goes off. \
+	Jobs: Chemist"
+	item = /obj/item/weapon/grenade/chem_grenade/clf3
+	cost = 11 //No way someone's getting more than one of these monsters

```

Comment:
it's out of 20, so either this remark was tongue in cheek or all the items you added are half the price you intended